src/uu/date/src/date.rs

```diff
@@ -115,7 +115,7 @@ impl<'a> From<&'a str> for Iso8601Format {
             NS => Self::Ns,
             DATE => Self::Date,
             // Should be caught by clap
-            _ => panic!("Invalid format: {s}"),
+            _ => panic!("date:` invalid argument `{s}` for '--iso-8601'"),
```

Comment:
maybe 
```suggestion
            _ => unreachable!(),
```
if we won't go there then

Reply:
ping ?

Reply:
Hey, this commit [`5e42b58` (#4499)](https://github.com/uutils/coreutils/pull/4499/commits/5e42b58927a19d26c5df01640e59f4c6426539e3) fixes it.

Still missing the unit test. Will ping when I have it working.

Reply:
thanks